frontend/src/tbpro/elements/TextInput.vue

```diff
@@ -167,6 +167,7 @@ const onChange = () => {
     overflow: hidden;
     max-width: 40%;
     text-wrap: nowrap;
+    z-index: 1;
```

Comment:
For some weird reason the prefix element gets hidden behind the input as soon as that gets disabled 🤷🏻